subsys/net/ip/net_context.c

```diff
@@ -2198,6 +2320,16 @@ static int set_context_sndtimeo(struct net_context *context,
 #endif
 }
 
+#if IS_ENABLED(CONFIG_NET_TCP_KEEPALIVE)
+static int set_context_keepalive(struct net_context *context,
+				const void *value, size_t len)
+{
+	context->options.keepalive = (*(const int *)value)?true:false;
```

Comment:
This will probably lead to checkpatch warnings, could you add more spaces here so something like
```
context->options.keepalive = *(const int *)value ? true : false;
```